testpilot/experiments/models.py

```diff
@@ -11,7 +13,9 @@
 from ..utils import HashedUploadTo
 
 
+experiment_facebook_upload_to = HashedUploadTo('image')
```

Comment:
Did just notice here, though, that these social media images are required fields. So, the fallback works for existing experiments without error. But, future edits to experiments will need *something* plugged into the image fields.

More a caveat to note than a nit to change, IMO, because we really *do* want images here.